### PR TITLE
Add lecli as user agent (with version information if available)

### PR DESCRIPTION
**Pull request by scawley-r7**
@StephenHynes7 @stopal-r7 @jfitzpatrick-r7

### CODE REVIEW

File: lecli/lecli.py [hunk 1]
```diff
@@ -4,9 +4,10 @@
 import queryapi
 import userapi
 
-
+VERSION = 0.1
```
Comment (stopal-r7) [inline lecli/lecli.py]: I think it is better if we follow PEP8 naming convention for versioning: http://stackoverflow.com/a/459185

File: lecli/lecli.py [hunk 2]
```diff
@@ -4,9 +4,10 @@
...
+apiutils.VERSION = VERSION
```
Comment (stopal-r7) [inline lecli/lecli.py]: Why don't we store version number in one place? Importing a module containing version (like _version.py) won't do any harm to modularity.

I like the way its implemented in twisted:
https://github.com/powdahound/twisted/blob/master/twisted/_version.py
https://github.com/powdahound/twisted/blob/master/twisted/__init__.py

**Comment by stopal-r7** [comment]
I think it's a great idea, being able to measure usage of clients like lecli and leexportpy. What do you think @jfitzpatrick-r7?

**Comment by StephenHynes7** [comment]
Lets decided on the versioning otherwise LGTM

**Comment by scawley-r7** [comment]
I've updated this to just modify the user agent header. Maybe versioning is a discussion for another PR?

**Comment by stopal-r7** [comment]
LGTM

**Pull request merged**